app/ante/eip712.go

```diff
@@ -217,7 +223,7 @@ func VerifySignature(
 			return errorsmod.Wrap(errortypes.ErrNoSignatures, "tx doesn't contain any msgs to verify signature")
 		}
 
-		txBytes := legacytx.StdSignBytes(
+		txBytes := eip712.ConstructUntypedEIP712Data(
```

Comment:
I saw a comment that tx timeout was disable because it wasn't passed through, but appears it is included in the sign doc?